keymanapp/keyman
# fix(web): Fix clearing of deadkeys

- don't clear deadkeys after pressing backspace
- add unit test
- also some small refactorings

Fixes #8568.

# User Testing

**TEST_8568**: Verify that #8568 is fixed

1. Type `phaa saa laaw ko` then press <kbd>bksp</kbd>, <kbd>bksp</kbd>
2. Verify you get the following output: ພາສາລາວ
